<!--
  1 failure:  :disappointed_rel...
  1 warning:  There is no linke...
  2 messages
  
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  :disappointed_relieved: `npm test` failed. See below for details.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  There is no linked JIRA ticket in the PR body. Please include the URL of the relevant JIRA ticket. If you need to, you may file a ticket on <a href="https://jira.appcelerator.org/secure/CreateIssue!default.jspa">JIRA</a>
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  :thumbsup: Hey!, You deleted more code than you added. That's awesome!
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  ```
> titanium-mobile@7.2.0 test /Users/build/jenkins/workspace/dk_titanium_mobile_PR-10047-MOEICTRP3BN65FWLGCLKFEVQVZR4KJ3PKKQ5JHUP4QZIAX24UQVQ
> grunt

Running "appcJs:src:lintOnly" (appcJs) task

Running "eslint:src" (eslint) task

/Users/build/jenkins/workspace/dk_titanium_mobile_PR-10047-MOEICTRP3BN65FWLGCLKFEVQVZR4KJ3PKKQ5JHUP4QZIAX24UQVQ/tests/Resources/ti.ui.window.addontest.js
  38:3  error  'didFocus' is not defined  no-undef

✖ 1 problem (1 error, 0 warnings)

Warning: Task "eslint:src" failed. Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.

```
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

--- a/tests/Resources/ti.ui.window.addontest.js
+++ b/tests/Resources/ti.ui.window.addontest.js
@@ -34,10 +34,6 @@ describe('Titanium.UI.Window', function () {
 		rootWindow.close();
 	});
 
-	beforeEach(function () {
-		didFocus = false;
-	});
-
 	afterEach(function () {
 		if (win) {
 			win.close();

